reconcile/terraform_resources.py

```diff
@@ -207,13 +207,17 @@ def populate_oc_resources(spec, ri):
         logging.error(msg)
 
 
-def fetch_current_state(namespaces, thread_pool_size, internal, use_jump_host):
+def fetch_current_state(dry_run, namespaces, thread_pool_size, internal,
+                        use_jump_host):
     ri = ResourceInventory()
     settings = queries.get_app_interface_settings()
     oc_map = OC_Map(namespaces=namespaces, integration=QONTRACT_INTEGRATION,
```

Comment:
do we even need to initialize a real oc_map instance?

Reply:
Need for cleanup https://github.com/app-sre/qontract-reconcile/blob/master/reconcile/terraform_resources.py#L302